libpod/container_config.go

```diff
@@ -147,7 +147,7 @@ type ContainerRootFSConfig struct {
 	// Mounts contains all additional mounts into the container rootfs.
 	// It is presently only used for the container's SHM directory.
 	// These must be unmounted before the container's rootfs is unmounted.
-	Mounts []string `json:"mounts,omitempty"`
```

Comment:
This is going to break upgrades from previous versions - you can't deserialize a JSON string into a `define.InspectMount`. It should be possible to, instead of this, make a `func (c *Container) InspectMounts() ([]define.InspectMount, error)` that takes the libpod ContainerConfig and converts it into InspectMount in the same way Inspect does (with a minor refactor, you could use that instead). That would probably be sufficient for your needs here, no?

Reply:
>in the same way Inspect does (with a minor refactor, you could use the same code for both)
>
to which specific function you refer? Do you mean this one?
https://github.com/containers/podman/blob/490eb476a82e670ae3317e31411a8288215eb6df/libpod/container_inspect.go#L39